frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx

```diff
@@ -51,13 +55,35 @@ class PopinFixedHeader extends React.Component {
   }
 
   render () {
-    const { customClass, customColor, faIcon, rawTitle, componentTitle, userRoleIdInWorkspace, onClickCloseBtn, disableChangeTitle, showChangeTitleButton, t } = this.props
-    const { state } = this
+    const { state, props } = this
+    const {
```

Comment:
I'm unsure whether destructuring `props` is an accepted style (but I personally do not have an issue with it).

Reply:
Personally I prefer leave `props.`, but since this component was already like that, we choose follow the style.